CRM/Report/Form.php

```diff
@@ -3692,11 +3694,7 @@ public function buildGroupTempTable() {
         WHERE smartgroup_contact.group_id IN ({$smartGroups}) ";
     }
 
-    $this->groupTempTable = CRM_Utils_SQL_TempTable::build()->setCategory('rptgrp')->setId(date('Ymd_') . uniqid())->getName();
```

Comment:
this does change the naming convention a little - but it wasn't really deliberate before so that seems fine